### ISSUE TITLE
Add “exit” button to main menu

### ISSUE THREAD

**Issue by Wuzzy2** [minetest/minetest#2426]
The main menu of Minetest is currently lacking a button to exit Minetest. I wish Minetest had such a button.

I suggest to create a simple button on the upper right part of the screen, so it is not on the same space as the main “window” and is always visible, no matter which tab is opened.

**Comment by rubenwardy** [comment]
We could use a more game like menu too. Rather than ugly tabs.

**Comment by MT-Modder** [comment]
What's the difference between closing the window and closing via the close button?

**Comment by rubenwardy** [comment]
Full screen. You could use the escape, but not as obvious.

**Comment by MT-Modder** [comment]
I didn't know minetest had "full" fullscreen. Last time I tried it just maximized the window. Is full screen OS dependent?

**Comment by LeMagnesium** [comment]
With fullscreen enabled, you don't have the menu bar in the top of your window, neither its borders. These are as far as I know the only differences.
An exit button could be useful indeed, but where should we place it in the main menu?

**Comment by MT-Modder** [comment]
So the taskbar is still present? That's probably why I never noticed a difference.

**Comment by C1ffisme** [comment]
Cool! Minetest has a 'full' fullscreen?

Why are we going to put an exit button in the upper right corner? Most people look for it at the bottom of the menu.

**Comment by est31** [comment]
If the exit button should come to the upper right corner, it should only be displayed in full screen mode. Otherwise, we can expect that the user has a way to close minetest.

If the exit button should be displayed all times, it should better go into one of the tabs.

**Comment by MT-Modder** [comment]
Full screen doesn't do anything for me. It's F12 right? I press it and nothing happens.

**Comment by C1ffisme** [comment]
No, F11 is usually the fullscreen key.

**Comment by MT-Modder** [comment]
That's right F12 is screenshot... *facepalm*. Thanks.

**Comment by C1ffisme** [comment]
(shrug) Works for me...

**Comment by MT-Modder** [comment]
What OS do you use?

**Comment by PilzAdam** [comment]
Removed offtopic comments about fullscreen.

**Comment by paramat** [comment]
Any more comments?
Is anyone interested in coding this sometime? if not closing in 1 month.

**Comment by Wuzzy2** [comment]
I don't know what else to say … a close button is a very obvious feature, don't you think?

I don't think it is a good idea to close all old issues only because they're old. Please don't kill feature requests on the basis that nobody coded them *yet*. And this thing will probably pop up again anyway.
Is there a reason why this feature is not desired?

**Comment by paramat** [comment]
Yes a good idea, but if after 2.5 years no-one has stated an interest in coding it ...

**Comment by C1ffisme** [comment]
_It's literally just a button that tells the program to exit,_ are we just not interested in coding it because of higher-priority work, or do we just shove smaller requests under the table just to save time?

**Comment by paramat** [comment]
Maybe Wuzzy2 can code it then? To a large degree in Minetest, if you want something done you have to do it yourself.

**Comment by paramat** [comment]
@rubenwardy i can't tell if you support this .. ?

**Comment by mk-pmb** [comment]
@p5yk0g3n3s1s
> What's the difference between closing the window and closing via the close button?

User intent. Reassuring the user. Clarity about what the click will (not) do. #10894
